<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLabel" name="label_title_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>991</x>
     <y>17</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_max">
   <property name="geometry">
    <rect>
     <x>968</x>
     <y>17</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_min">
   <property name="geometry">
    <rect>
     <x>943</x>
     <y>17</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mini">
   <property name="geometry">
    <rect>
     <x>922</x>
     <y>17</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>128</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_shuxian">
   <property name="geometry">
    <rect>
     <x>907</x>
     <y>17</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set">
   <property name="geometry">
    <rect>
     <x>872</x>
     <y>17</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_message">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>17</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_skin">
   <property name="geometry">
    <rect>
     <x>788</x>
     <y>17</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_message_Tip">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_people">
   <property name="geometry">
    <rect>
     <x>696</x>
     <y>18</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_people">
   <property name="geometry">
    <rect>
     <x>696</x>
     <y>18</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>梦想少年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_people_1">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>12</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_title_left">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>14</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_title_right">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>14</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title_input">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>14</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>16</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_input_back">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>14</y>
     <width>220</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_bottom_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>1022</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>628</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>630</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_last">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>630</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>640</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_music">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>640</y>
     <width>437</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_long">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>640</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_horn">
   <property name="geometry">
    <rect>
     <x>748</x>
     <y>639</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_sound">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>646</y>
     <width>100</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_circulation">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>639</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lyric">
   <property name="geometry">
    <rect>
     <x>926</x>
     <y>639</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_list">
   <property name="geometry">
    <rect>
     <x>959</x>
     <y>638</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sound_count">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>638</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>111</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_part">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>1</width>
     <height>570</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_list" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>200</width>
     <height>401</height>
    </rect>
   </property>
   <widget class="QListWidget" name="listWidget_mymusic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_mymusic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 我的音乐</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_recommend">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 推荐</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>200</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_Create_form" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_unfold">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_text_create_music_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>3</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string> 创建的歌单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>8</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_text_create_music_list</zorder>
    <zorder>pushButton_add</zorder>
    <zorder>pushButton_unfold</zorder>
   </widget>
   <widget class="QWidget" name="widget_Collect_form" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="pushButton_unfold_collect">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_text_create_music_list_collect">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>3</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string> 收藏的歌单</string>
     </property>
    </widget>
    <zorder>label_text_create_music_list_collect</zorder>
    <zorder>pushButton_unfold_collect</zorder>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_main">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>822</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_tuijian">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>个性推荐</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gedan">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>401</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>歌单</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_zhubodiantai">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主播电台</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_paihangbang">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>排行榜</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_geshou">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>歌手</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_zuixinyinyue">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最新音乐</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_line">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>82</y>
     <width>776</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 34, 38);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_title_back</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>pushButton_max</zorder>
  <zorder>pushButton_min</zorder>
  <zorder>pushButton_mini</zorder>
  <zorder>label_title_logo</zorder>
  <zorder>label_shuxian</zorder>
  <zorder>pushButton_set</zorder>
  <zorder>pushButton_message</zorder>
  <zorder>pushButton_skin</zorder>
  <zorder>label_message_Tip</zorder>
  <zorder>label_people</zorder>
  <zorder>pushButton_people</zorder>
  <zorder>pushButton_people_1</zorder>
  <zorder>pushButton_title_left</zorder>
  <zorder>pushButton_title_right</zorder>
  <zorder>label_input_back</zorder>
  <zorder>lineEdit_title_input</zorder>
  <zorder>pushButton_search</zorder>
  <zorder>label_bottom_back</zorder>
  <zorder>pushButton_play</zorder>
  <zorder>pushButton_next</zorder>
  <zorder>pushButton_last</zorder>
  <zorder>label_time</zorder>
  <zorder>horizontalSlider_music</zorder>
  <zorder>label_time_long</zorder>
  <zorder>pushButton_horn</zorder>
  <zorder>horizontalSlider_sound</zorder>
  <zorder>pushButton_circulation</zorder>
  <zorder>pushButton_lyric</zorder>
  <zorder>label_sound_count</zorder>
  <zorder>pushButton_list</zorder>
  <zorder>label_part</zorder>
  <zorder>widget_list</zorder>
  <zorder>label_line</zorder>
  <zorder>tabWidget_main</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
